common/db.go

```diff
@@ -111,9 +115,27 @@ var schema = `
 		sig BLOB,
 		sessionID STRING
 	);
-
 	CREATE INDEX IF NOT EXISTS idx_winningtickets_sessionid ON winningTickets(sessionID);
 
+	CREATE TABLE IF NOT EXISTS ticketQueue (
+		createdAt DATETIME DEFAULT CURRENT_TIMESTAMP,
+		sender STRING,
+		recipient STRING,
+		faceValue BLOB,
+		winProb BLOB,
+		senderNonce INTEGER,
+		recipientRand BLOB,
+		recipientRandHash STRING,
+		sig BLOB PRIMARY KEY,
+		creationRound int64,
+		creationRoundBlockHash STRING,
+		paramsExpirationBlock int64,
+		submittedAt DATETIME,
```

Comment:
Minor question: perhaps `redeemedAt` would be more descriptive since we're setting this timestamp after the ticket has been redeemed?

Reply:
fixed in 4df5f0d769d1bb242c14b39c1fb37bd4165765db